gulpfile.js

```diff
@@ -600,14 +598,10 @@ function buildExtensionJs(path, name, version, options) {
     throw new Error('Unsupported loadPriority: ' + options.loadPriority);
   }
   const priority = options.loadPriority ? 'p:"high",' : '';
-  return compileJs(path + '/', filename, './dist/v0', {
-    watch: options.watch,
-    preventRemoveAndMakeDir: options.preventRemoveAndMakeDir,
-    minify: options.minify,
+  return compileJs(path + '/', filename, './dist/v0', Object.assign(options, {
     toName: name + '-' + version + '.max.js',
```

Comment:
If you're moving this over to Object.assign perhaps it's also worth reformatting these to template literals.

```javascript
Object.assign(options, {
  toName: `${name}-${version}.max.js`,
  minifiedName: `${name}-{version}.js`,
  latestName: `${name}-latest.js`,
  wrapper: options.noWrapper ? '' : `(self.AMP=self.AMP||[]).push({n:"${name}",${priority}v:"${internalRuntimeVersion}",f:(function(AMP){<%= contents %>\n})});`
})
```